### PR TITLE
Enable Active-Active gateway feature operations.

### PR DESCRIPTION
**Pull request by Nilambari**
Changes in files:-
1. Generated/Models/VirtualNetworkGateway.cs :- Added Active-Active feature flag
2. Properties/AssemblyInfo.cs and project.json :- Network package version increment.
3. GatewayOperationsTests.cs :- Added new test: VirtualNetworkGatewayActiveActiveFeatureTest to validate enabled Active-Active gateway feature related operations. Recorded and added this test result.
4. Most files have some typo fixes which got picked up in generated files.

### CODE REVIEW

**Comment by Nilambari** [comment]
@markcowl Can you Please approve this PR? Thanks!

File: src/ResourceManagement/Network/Microsoft.Azure.Management.Network/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -19,8 +19,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Network Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Network management functions for managing the Microsoft Azure Network service.")]
 
-[assembly: AssemblyVersion("6.1.0.0")]
-[assembly: AssemblyFileVersion("6.1.0.0")]
+[assembly: AssemblyVersion("6.1.1.0")]
```
Comment (markcowl) [inline src/ResourceManagement/Network/Microsoft.Azure.Management.Network/Properties/AssemblyInfo.cs]: Assembly version should be 6.0.0.0, to enable forward-compatible references to the dll

**Comment by markcowl** [comment]
@Nilambari.  One nit, otherwise LGTM

File: src/ResourceManagement/Network/Microsoft.Azure.Management.Network/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (Nilambari) [inline src/ResourceManagement/Network/Microsoft.Azure.Management.Network/Properties/AssemblyInfo.cs]: don't we need to increment the version? Should we change version from 6.1.0.0 to 6.0.0.0 in AssemblyInfo,cs and in project.json?

**Comment by Nilambari** [comment]
@markcowl  can you tell me where is nit so that I can fix it. Thnx!

**Comment by Nilambari** [comment]
@markcowl can you Please help accept this PR? Thanks!

**Pull request merged**